src/grid/types/formio-submission.ts

```diff
@@ -0,0 +1,17 @@
+export default interface FormioSubmission<T = any, stateType= state> {
```

Comment:
@vlad-shusterman FormioSubmission is not specific for grids only, I would move this interface to more root level because it can be used widely across whole angular-formio library